charts/argo-workflows/values.yaml

```diff
@@ -630,3 +630,12 @@ customArtifactRepository: {}
 #     passwordSecret:
 #       name: artifactory-creds
 #       key: password
+
+emissaryExecutor:
+  # -- The command/args for each image on workflow, needed when the command is not specified and the emissary executor is used.
+  ## See more: https://argoproj.github.io/argo-workflows/workflow-executors/#emissary-emissary
+  images: []
+  #  argoproj/argosay:v2:
+  #    cmd: [/argosay]
+  #  docker/whalesay:latest:
+  #    cmd: [/bin/bash]
```

Comment:
I wonder if there is better name.... 🤔 

*original config is below.
ref: https://argoproj.github.io/argo-workflows/workflow-controller-configmap.yaml
<img width="953" alt="image" src="https://user-images.githubusercontent.com/30188755/207616121-2291c520-72fb-4cf6-a74e-fc418c056631.png">

Reply:
maybe just `emissary`.  links are there in comments to give more info.  wdyt?

Reply:
Hi @jmeridth , thank you for your opinion! I will adopt your idea, below. 🙋 
```yaml
emissary:
  # -- The command/args for each image on workflow, needed when the command is not specified and the emissary executor is used.
  ## See more: https://argoproj.github.io/argo-workflows/workflow-executors/#emissary-emissary
  images: []
  #  argoproj/argosay:v2:
  #    cmd: [/argosay]
  #  docker/whalesay:latest:
  #    cmd: [/bin/bash]
```

Reply:
Hi @jmeridth , is there anything in particular I can do to get approved? 🙋